kubernetes/utils/create_from_yaml.py

```diff
@@ -54,35 +55,46 @@ def create_from_yaml(
         processing of the request.
         Valid values are: - All: all dry run stages will be processed
     """
+    if path.exists(yaml_file):
+        with open(path.abspath(yaml_file)) as f:
+            yaml_file = f.read()
+    else:
+        yaml_file = io.StringIO(yaml_file)
+
+    yml_document_all = yaml.safe_load_all(io.StringIO(yaml_file))
+    # Load all documents from a single YAML file
+    for yml_document in yml_document_all:
+        create_from_map(k8s_client, yml_document, verbose=verbose,
+                        **kwargs)
+
+
+def create_from_map(k8s_client, yml_document, verbose=False, **kwargs):
```

Comment:
If you do it this way, the verbose does not need a "False" to it. You can call directly from your previous function with ```create_from_map(k8s_client, yml_object, verbose, **kwargs)```

Reply:
I actually meant this to be public, so people can use it without a previous function. Did you meant to put this comment on line 64?

Reply:
Oh OK. Please import the function in ```__init__.py``` then.
I'm a bit leaning towards calling it ```create_from_dict``` since the hash maps (?) are dicts in Python. What do you think?

Reply:
Yeah, I guess we can do this. It's more pythonic.

Reply:
done.